person.rb

```diff
@@ -0,0 +1,35 @@
+require './nameable'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', _parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
```

Comment:
- [ ] Great job. please like the previous reviewer noted, Kindly initialize @rentals. Please check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for more guidance.